docs/canvas/canvas-tutorial.asciidoc

```diff
@@ -124,7 +124,7 @@ Here are some things to try:
 
 * Play with the {kibana-ref}/add-sample-data.html[sample Canvas workpads].
```

Comment:
What about instead of "Canvas Tutorial" using a title that is more goal-oriented.